tests/e2e_tests/astrom_e2e.py

```diff
@@ -178,7 +177,7 @@ def test_astrom_e2e(tvacdata_path, e2eoutput_path):
     assert dec == pytest.approx(target[1], abs=8.333e-7)
 
 if __name__ == "__main__":
-    tvacdata_dir = "/Users/macuser/Roman/corgi_contributions/Callibration_Notebooks/TVAC"
+    tvacdata_dir = "/Users/macuser/Roman/corgidrp_develop/calibration_notebooks/TVAC"
```

Comment:
Is there a standardized solution for what filepath to use for the data in the end-to-end tests so that it can be run on any machine?

Reply:
this is used only for debugging. the way the e2e tests are run (see README), don't use these lines of code.